lib/outgoing.js

```diff
@@ -259,7 +259,7 @@ function track(requestOptions, request, recorder, logger, moesifOptions) {
         )
       );
     }
-  }, 2000);
```

Comment:
If this is max cap for customer calls, let's do 30s. We don't want customers get confused by early termination